content/webapp/pages/exhibition-guide.tsx

```diff
@@ -223,9 +223,19 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
         res,
         req,
         maxAge: 8 * 60 * 60,
+        path: '/',
+        sameSite: false
```

Comment:
Is this temporary for debugging, or something we want permanently?

Reply:
I was going to leave the path permanently, but perhaps drop `sameSite` if we find that the context url is resolving to something expected. I'll add a note to these to remind myself. 

> 🦞
> 
> Looks good.
> 
> One suggestion: whenever I'm adding debugging logs, I often include a custom prefix like `@@AWLC`. It makes those lines easy to search for and makes it more obvious that those logs were written for a person to use for some one-off debugging, not part of the long-term logging approach.

Nice, that's a great suggestion, i'll do that